Makefile

```diff
@@ -49,11 +49,11 @@ test-only:
 test: lint test-only
 
 test-ci:
-	NODE_ENV=test make bootstrap
+	make bootstrap
 	make test-only
 
 test-ci-coverage:
-	NODE_ENV=test BABEL_ENV=cov make bootstrap
```

Comment:
Yeah that is odd, since BABEL_ENV is set here.

`const envKey = process.env.BABEL_ENV || process.env.NODE_ENV || "development";` in our code currently